DOCUMENTATION.md

```diff
@@ -797,6 +797,52 @@ If the request is not successful, the client returns an `Alphagov\Notifications\
 |`403`|`[{`<br>`"error": "AuthError",`<br>`"message": "Invalid token: signature, api token not found"`<br>`}]`|Use the correct API key. Refer to [API keys](#api-keys) for more information.|
 
 
+
+## Get a PDF for a letter notification
+
+### Method
+
+This returns the pdf contents of a letter notification.
+
+```php
+$pdf_file = $notifyClient->getPdfForLetter(
+  'f33517ff-2a88-4f6e-b855-c550268ce08a' // required string - notification ID
+)
+```
+
+### Arguments
+
+#### notificationId (required)
+
+The ID of the notification. You can find the notification ID in the response to the [original notification method call](/python.html#get-the-status-of-one-message-response).
+
+You can also find it in your [GOV.UK Notify Dashboard](https://www.notifications.service.gov.uk).
+
+1. Sign into GOV.UK Notify and select __Dashboard__.
+1. Select __letters sent__.
+1. Select the relevant notification.
+1. Copy the notification ID from the end of the page URL, for example `https://www.notifications.service.gov.uk/services/af90d4cb-ae88-4a7c-a197-5c30c7db423b/notification/ID`.
```

Comment:
We should signpost the _API integration_ page instead – it’s a friendlier way to get the ID:
![image](https://user-images.githubusercontent.com/355079/68143032-7c423800-ff28-11e9-9a3b-fca42eaa3193.png)